src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/OrganisationCreationRequest.java

```diff
@@ -38,6 +38,10 @@
     @NotNull
     private List<ContactInformationCreationRequest> contactInformation;
 
+    private final String dateReceived;
+
```

Comment:
These changes are not required as there are not new fields is coming up in request

Reply:
resolved